### PR TITLE
Added documentation for animated buttons

### PR DESCRIPTION
**Pull request by snapecode**
Added documentation for animated buttons to README and Documentation page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -192,6 +193,15 @@ Adding the `toggle-btn` class to your radio inputs converts them to toggle butto
 <input type="radio" class="sbtn toggle-btn" />
 ```
 
+## Animated Buttons
+
+To use Animated sButtons in your project, add the classes of the animated sButton you want to either `<button>` or `<a>` tags.
+
+```html
+<button class="sbtn atom-btn blue-btn">Animated Button</button>
```
Comment (shahednasser) [inline README.md:201]: you should add `animated` to the list of classes in the example.

File: website/src/pages/Documentation/index.js [hunk 2]
```diff
@@ -267,7 +268,22 @@ function Documentation () {
...
+                        </section>       
+
+                        <section id="animated-buttons" className="mt-7">
+                            <SubsectionHeader title="Animated Buttons" to="animated-buttons" />
+                            <div className="pl-8 text-lg">
+                                <p className="mb-3">
+                                    To use Animated sButtons in your project, add the classes of the animated sButton you want to either <code>{`<button>`}</code> or <code>{`<a>`}</code> tags.<br />
+                                    For example:
+                                </p>
+                                <Code code={`<button class="sbtn atom-btn blue-btn">Animated Button</button>`} language="html" />
```
Comment (shahednasser) [inline website/src/pages/Documentation/index.js:280]: The class `animated` should be added to this code + the example below it

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you for your work @snapecode please find my comments below

**Comment by snapecode** [comment]
Got the animated classes in, how is the description in the readme? could it use more info on using the mixins?

**Comment by shahednasser** [comment]
Great work thank you!

@all-contributors please add @snapecode for documentation

**Pull request merged**